gt_concordance/gt_concordance.py

```diff
@@ -0,0 +1,524 @@
+        click.echo('Analysis path does not exist! Path: {}'.format(ANALYSIS_PATH))
+        click.echo('Terminating')
+        exit(0)
+    config.update({'ANALYSIS_PATH': ANALYSIS_PATH})
+
+    GATK_PATH = config.get('GATK_PATH')
+    if GATK_PATH is None:
+        click.echo("config file missing GATK_PATH")
+        click.echo('Do you want to enter path to GATK? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            click.echo('Please enter path to GATK')
+            GATK_PATH = raw_input()
+        else:
+            exit(0)
+    if not os.path.exists(GATK_PATH):
+        click.echo('GATK file does not exist! Path: {}'.format(GATK_PATH))
+        click.echo('Terminating')
+        exit(0)
+    config.update({'GATK_PATH': GATK_PATH})
+
+    GATK_REF_FILE = config.get('GATK_REF_FILE')
+    if GATK_REF_FILE is None:
+        click.echo("config file missing GATK_REF_FILE")
+        click.echo('Do you want to enter path to reference file? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            click.echo('Please enter path to reference file')
+            GATK_REF_FILE = raw_input()
+        else:
+            exit(0)
+    if not os.path.exists(GATK_REF_FILE):
+        click.echo('Reference file does not exist! Path: {}'.format(GATK_REF_FILE))
+        click.echo('Terminating')
+        exit(0)
+    config.update({'GATK_REF_FILE': GATK_REF_FILE})
+
+    GATK_VAR_FILE = config.get('GATK_VAR_FILE')
+    if GATK_VAR_FILE is None:
+        click.echo("config file missing GATK_VAR_FILE")
+        click.echo('Do you want to enter path to GATK variant file? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            click.echo('Please enter the path to GATK variant file')
+            GATK_VAR_FILE = raw_input()
+        else:
+            exit(0)
+    if not os.path.exists(GATK_VAR_FILE):
+        click.echo('GATK variant file does not exist! Path: {}'.format(GATK_VAR_FILE))
+        click.echo('Terminating')
+        exit(0)
+    config.update({'GATK_VAR_FILE': GATK_VAR_FILE})
+
+    INTERVAL_FILE = config.get('INTERVAL_FILE')
+    if INTERVAL_FILE is None:
+        click.echo('config file missing INTERVAL_FILE')
+        click.echo('Do you want to enter path to interval file? Y/n')
+        to_enter = raw_input()
+        if to_enter.lower() == 'y':
+            click.echo('Please enter the path to interval file')
+            INTERVAL_FILE = raw_input()
+        else:
+            exit(0)
+    if not os.path.exists(INTERVAL_FILE):
+        click.echo('Interval file does not exist! Path: {}'.format(INTERVAL_FILE))
+        click.echo('Terminating')
+        exit(0)
+    config.update({'INTERVAL_FILE': INTERVAL_FILE})
+
+    return True
+
+def parse_vcf_file(sample, config):
+    project = sample.split('_')[0]
+    path = os.path.join(config.get('ANALYSIS_PATH'), project, 'piper_ngi/03_genotype_concordance', '{}.vcf'.format(sample))
```

Comment:
clearly in this context the vcf file needs to exists because you have guaranteed the existence of it before calling the function, but as a code rule I would say that if a function guesses a name from the path than the function is responsible to handle the fact that the file do not exits. In this case you can pass to this function the file itself